spec/datadog/tracing/contrib/graphql/test_schema_examples.rb

```diff
@@ -134,4 +151,37 @@ def unload_test_schema(prefix: '')
       end
     end
   end
+
+  describe 'query with a GraphQL error' do
+    subject(:result) { schema.execute(query: 'query Error{ graphqlError }', variables: { var: 1 }) }
+
+    let(:graphql_execute) { spans.find { |s| s.name == 'graphql.execute' } }
+
+    it 'creates query span for error' do
+      expect(result.to_h['errors'][0]['message']).to eq('GraphQL error')
+      expect(result.to_h['data']).to eq('graphqlError' => nil)
+
+      expect(graphql_execute.resource).to eq('Error')
+      expect(graphql_execute.service).to eq(service)
+      expect(graphql_execute.type).to eq('graphql')
+
+      expect(graphql_execute.get_tag('graphql.source')).to eq('query Error{ graphqlError }')
+
+      expect(graphql_execute.get_tag('graphql.operation.type')).to eq('query')
+      expect(graphql_execute.get_tag('graphql.operation.name')).to eq('Error')
+
+      expect(graphql_execute.events).to contain_exactly(
+        a_span_event_with(
+          name: 'dd.graphql.query.error',
+          attributes: {
+            'message' => 'GraphQL error',
+            'type' => 'GraphQL::ExecutionError',
+            'stacktrace' => include(__FILE__),
+            'locations' => ['1:14'],
+            'path' => ['graphqlError'],
+          }
+        )
+      )
+    end
+  end
```

Comment:
Maybe worth perhaps adding a test for multiple errors? Since that seems a big part of the feature

Reply:
Added!